Cargo.toml

```
@@ -103,6 +103,10 @@ path = "examples/2d/contributors.rs"
 name = "mesh"
 path = "examples/2d/mesh.rs"
 
+[[example]]
+name="move_sprite"
+path="examples/2d/move_sprite.rs"
```

The CI (https://github.com/bevyengine/bevy/pull/2391/checks?check_run_id=2909210823) doesn't seem to realize that example is present here. 🤔 
Perhaps the extra spaces are necessary here
```suggestion
[[example]]
name = "move_sprite"
path = "examples/2d/move_sprite.rs"
```